package.json

```diff
@@ -279,6 +279,26 @@
                 },
                 "enablement": "notebookViewType == jupyter-notebook && jupyter.isnotebooktrusted"
             },
+            {
+                "command": "jupyter.notebookEditor.runallcellsabove",
```

Comment:
From what I can see we don't camelCase any other command names. So should keep "notebookeditor" here for consistancy.